src/index.js

```diff
@@ -0,0 +1,31 @@
+import './style.css';
+import fetchData from './modules/getData.js';
+import postData from './modules/postData.js';
+
+const refresh = document.querySelector('.refresh-btn');
+const form = document.getElementById('form');
+const userInput = document.getElementById('user');
+const scoreInput = document.getElementById('score');
+const text = document.querySelector('.text');
+
+// submit form
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  postData();
+  userInput.value = '';
+  scoreInput.value = '';
+});
```

Comment:
[OPTIONAL] I think it is a good idea to add validation to prevent the user try to submit blank entries. It helps to improve the user's experience. :eyes: :eyes:  You can also add a message to inform about this the user.